06.bowling_object/lib/game.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  attr_reader :frames
+
+  STRIKE_MARK = 'X'
+
+  def initialize(shot_marks)
+    @frames = []
+    shot_pins = shot_marks.split(',').map do |mark|
+      convert_to_i(mark)
+    end
+    build_frames(shot_pins)
+  end
+
+  def score
+    pin_score = @frames.sum(&:score)
+    strike_bonus = calculate_strike_bonus
+    spare_bonus = calculate_spare_bonus
+    pin_score + strike_bonus + spare_bonus
+  end
+
+  private
+
+  def convert_to_i(mark)
+    mark == STRIKE_MARK ? 10 : mark.to_i
+  end
+
+  def build_frames(shot_pins)
+    shot_pins.each do |pins|
```

Comment:
pinsは配列？（複数形なので）

あと、

```suggestion
    shot_marks.split(',').map do |mark|
      pins = convert_to_i(mark)
```

みたいになってる方がループの回数が減って効率が良いと思います

Reply:
Frame内のpinは1〜3つとなるので配列（複数形）にしています。